flight/move_to_test.py

```diff
@@ -0,0 +1,94 @@
+"""
+Main driver code for moving drone to each waypoint
+"""
+
+import asyncio
+import goto
+from mavsdk import System
+import logging
+from typing import List
```

Comment:
Don't need to import List from typing in python 3.10+. You can type lists like this without any imports: `var: list[int] = 1`